.mise.toml

```diff
@@ -149,33 +158,66 @@ docker push $IMAGE:latest
 docker push $IMAGE:$VERSION
 """
 
+[tasks."discord:build"]
+description = "build docker image for discord bot"
+run = """
+cd $root/exec/infrastructure/discord
+
+IMAGE=pocketsizefund/discord
+VERSION=$(poetry version -s)
+
+docker build -t $IMAGE:latest -t $IMAGE:$VERSION --platform $PLATFORM .
+"""
+
+[tasks."discord:run"]
+description = "run price model in docker compose"
+run = """
+cd $root/exec/infrastructure/pricemodel
+docker compose up --build
+"""
+
+[tasks."discord:push"]
```

Comment:
Is there a reason these two tasks are prefixed with `discord`?

Reply:
this is the service to send discord events, so it's like a command for building and a command for pushing, similar to how we have multiple `price-model:*` or `positionmanager:*` commands